packages/cli/create-strapi-app/src/prompts.ts

```diff
@@ -22,12 +22,57 @@ async function typescript() {
     {
       type: 'confirm',
       name: 'useTypescript',
-      message: 'Do you want to use Typescript ?',
+      message: 'Start with Typescript?',
       default: true,
     },
   ]);
 
   return useTypescript;
 }
 
-export { directory, typescript };
+async function example() {
+  const { useExampleApp } = await inquirer.prompt<{
+    useExampleApp: boolean;
+  }>([
+    {
+      type: 'confirm',
+      name: 'useExampleApp',
+      message: 'Start with an example structure & data?',
+      default: true,
```

Comment:
Is the onboarding enabled in the example app? If not, won't that create confusion for new users? (how to create a CT, create content, manage permissions...)

Reply:
it's always enabled on new app

Reply:
Nice, I thought it was based on whether you already add content-types & co